src/Provider/en_US/Restaurant.php

```diff
@@ -4,7 +4,6 @@
 
 class Restaurant extends \Faker\Provider\Base
 {
-
```

Comment:
The file did not have significant changes, it is not necessary to be sent, you must remove the modified line.